crates/matrix-sdk/src/send_queue/mod.rs

```diff
@@ -1841,6 +1841,10 @@ pub enum RoomSendQueueError {
     /// Error coming from storage.
     #[error(transparent)]
     StorageError(#[from] RoomSendQueueStorageError),
+
+    /// The attachment event failed to be created.
+    #[error("the attachment event could not be created")]
+    AttachmentError,
```

Comment:
nit: since it's not a transparent error type, let's not use the `Error` suffix, and rename it to `FailedToCreateAttachment`